app/src/main/java/net/gini/android/gvlexample/results/ResultsContract.java

```diff
@@ -0,0 +1,28 @@
+package net.gini.android.gvlexample.results;
+
+import net.gini.android.gvlexample.PresenterContract;
+import net.gini.android.gvlexample.ViewContract;
+import net.gini.android.gvlexample.gini.Extraction;
+
+import java.util.List;
+
+/**
+ * Created by aszotyori on 23.11.17.
+ */
+
+public abstract class ResultsContract {
```

Comment:
When I was coding in Android, I loved this kind of classes because they gave me a quick look about a screen❤️